aws/resource_aws_media_convert_queue_test.go

```diff
@@ -0,0 +1,365 @@
+package aws
+
+import (
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/mediaconvert"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+)
+
+func TestAccAWSMediaConvertQueue_basic(t *testing.T) {
+	var queue mediaconvert.Queue
+	resourceName := "aws_media_convert_queue.test"
+	rName := acctest.RandomWithPrefix("tf-acc-test")
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAwsMediaConvertQueueDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccMediaConvertQueueConfig_Basic(rName),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckAwsMediaConvertQueueExists(resourceName, &queue),
+					testAccMatchResourceAttrRegionalARN(resourceName, "arn", "mediaconvert", regexp.MustCompile(`queues/.+`)),
+					resource.TestCheckResourceAttr(resourceName, "name", rName),
+					resource.TestCheckResourceAttr(resourceName, "pricing_plan", mediaconvert.PricingPlanOnDemand),
+					resource.TestCheckResourceAttr(resourceName, "status", mediaconvert.QueueStatusActive),
+				),
+			},
+			{
+				ResourceName:      resourceName,
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func TestAccAWSMediaConvertReservedQueue_basic(t *testing.T) {
```

Comment:
Acceptance test function names should have a consistent prefix for a resource so they can all be run together, e.g. `TestAccAWSMediaConvertQueue_`

```suggestion
func TestAccAWSMediaConvertQueue_ReservationPlanSettings(t *testing.T) {
```